pkgs/development/libraries/mesa/default.nix

```diff
@@ -46,6 +46,13 @@ let
     optionals stdenv.isLinux (if (stdenv.isAarch32 || stdenv.isAarch64)
     then []
     else ["intel"] ++ lib.optional enableRadv "radeon");
+  llvm = llvmPackages.llvm.override {
```

Comment:
You can do
```
. override (old: {
  enableTargets = old.enableTargets ++ [ "AMDGPU" ]; 
})
```

Reply:
That does not seem to work for me: attribute 'enableTargets' missing

Reply:
Since mesa is a common package many users will end up with two llvm versions.
For that reason, I would propose to include that target by default.

Reply:
in `pkgs/development/compilers/llvm/7/llvm.nix` I would add ` enableTargets ? [ stdenv.hostPlatform stdenv.targetPlatform "AMDGPU" ]`

Reply:
Ideally with a comment why.

Reply:
For consistency we can add it to both but keep llvm6 since as default for mesa since we also use it as default in other places.

Reply:
For the next release we could look in to making llvm7 the new default since we depend on llvm7 with rustc anyway.